sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -737,6 +737,16 @@ def _add_argparse_args(cls, parser):
               '"<process to execute>", "env":{"<Environment variables 1>": '
               '"<ENV_VAL>"} }. All fields in the json are optional except '
               'command.'))
+    parser.add_argument(
+        '--sdk-worker-parallelism', default=None,
+        help=('Sets the number of sdk worker processes that will run on each '
+              'worker node. Default is 1. If 0, it will be automatically set '
+              'by the runner by looking at different parameters (e.g. number '
+              'of CPU cores on the worker machine).'))
+    parser.add_argument(
+        '--environment-cache-millis', default=0,
```

Comment:
Is 0 a reasonable default? It causes Portable wordcount tests to fail, unless we set a different value. In our gradle files we set 10000.

Reply:
0 should be fine. We should update the test to set this value if the test fails.